src/option-types/coap_option_cf.c

```diff
@@ -24,26 +24,46 @@
 //add content-format option
 CoAP_Result_t _rom AddCfOptionToMsg(CoAP_Message_t* msg, uint16_t cf)
```

Comment:
I would even remove the old method. There is no point in being backward compatible here, since we do not even have a stable release.